core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -416,13 +416,18 @@ private void startInstance() {
             this.geyserServer = new GeyserServer(this, bedrockThreadCount);
             this.geyserServer.bind(new InetSocketAddress(config.getBedrock().address(), config.getBedrock().port()))
                 .whenComplete((avoid, throwable) -> {
+                    String address = config.getBedrock().address();
+                    String port = String.valueOf(config.getBedrock().port()); // otherwise we get commas
+
                     if (throwable == null) {
-                        logger.info(GeyserLocale.getLocaleStringLog("geyser.core.start", config.getBedrock().address(),
-                                String.valueOf(config.getBedrock().port())));
+                        if ("0.0.0.0".equals(address)) {
+                            // basically just hide it in the log because some people get confused and try to change it
+                            logger.info(GeyserLocale.getLocaleStringLog("geyser.core.start.ip_supressed", port));
```

Comment:
`supressed`

Reply:
WAIT OOPS `suppressed`

Reply:
https://github.com/GeyserMC/languages/commit/47e827c5fc2854c967d69af2fdc2f3dd5ad811f7
Unfortunately, that typo was done months ago by me in the PR to the languages repo :/
I could PR a fix, but I'm unsure about whether that would remove all existing transactions for it

Reply:
Ouch - we might be able to find-replace all language files and that will work?

Reply:
I've done renaming on crowdin with zero issues

Reply:
i would probably recommended that because everything apart from en_US should be managed/edited by crowdin (right?)

Reply:
nevermind, apparently that didn't go well for me lol

Reply:
https://github.com/GeyserMC/languages/issues/135
Fixed by rtm, just needs to be merged and updated here